js-exercises/alphabetic-shift/alphabeticShift.js

```diff
@@ -0,0 +1,11 @@
+const alphabeticShift = input => {
+  if (input && input.length) {
+    return input
+      .split('')
+      .map(char => String.fromCharCode(char.charCodeAt(0) + 1))
```

Comment:
Did you consider what happens when a given string has a letter `z` or `Z`?

Reply:
Here It is not required to convert it to an array and use map instead you can use for of loop for string directly.